src/modules/site-v2/templates/about/about.html

```diff
@@ -133,50 +158,129 @@ <h2>CaeNDR goals</h2>
-    ACCESS_TOKEN = mapboxgl.accessToken;
-    var markerHeight = 50, markerRadius = 10, linearOffset = 25;
-    var popupOffset = {
-      'top': [0, 0],
-      'top-left': [0,0],
-      'top-right': [0,0],
-      'bottom': [0, -markerHeight],
-      'bottom-left': [linearOffset, (markerHeight - markerRadius + linearOffset) * -1],
-      'bottom-right': [-linearOffset, (markerHeight - markerRadius + linearOffset) * -1],
-      'left': [markerRadius, (markerHeight - markerRadius) * -1],
-      'right': [-markerRadius, (markerHeight - markerRadius) * -1]
-    };
-
-  data = {{ strain_listing|tojson|safe }}
+
+    // Icons
+{% autoescape off %}
+const icon_elegans = L.icon({
+    iconUrl: '{{ ext_asset("img/icons/elegansMarkerOutline.svg") }}',
+    iconSize: [20, 30],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+
+const icon_briggsae = L.icon({
+    iconUrl: '{{ ext_asset("img/icons/briggsaeMarkerOutline.svg") }}',
+    iconSize: [20, 30],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+const icon_tropicalis = L.icon({
+    iconUrl: '{{ ext_asset("img/icons/tropicalisMarkerOutline.svg") }}',
+    iconSize: [20, 30],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+
+
+const icon_hover = L.icon({
+    iconUrl: '{{ ext_asset("img/icons/hoverMarkerOutline.svg") }}',
+    iconSize: [20, 30],
+    iconAnchor: [12, 16],
+    popupAnchor: [0, -15],
+});
+{% endautoescape %}
+
+const MB_ATTR = 'Tiles &copy; Esri &mdash; Source: Esri, DeLorme, NAVTEQ, USGS, Intermap, iPC, NRCAN, METI, TomTom, 2012';
+const MB_URL = 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}';
+const OSM_URL = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+const OSM_ATTRIB = '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap</a> contributors';
+
+const southWest = L.latLng(-100, -200);
+const northEast = L.latLng(100, 200);
+const bounds = L.latLngBounds(southWest, northEast);
+
+const map = L.map('map', {minZoom:1, maxZoom:8, maxBounds: bounds, scrollWheelZoom: false, zoomControl: false, attributionControl: false,}).setView([20, -100], 1);
+L.tileLayer(MB_URL, {attribution: MB_ATTR, id: 'mapbox.streets', continuousWorld: false, worldCopyJump: true}).addTo(map);
+
+const Species = Object.freeze({
+  ELEGANS: 'Caenorhabditis elegans',
+  BRIGGSAE: 'Caenorhabditis briggsae',
+  TROPICALIS: 'Caenorhabditis tropicalis'
+});
+
+$(document).ready(function () {
+
+  const data = {{ strain_listing|tojson|safe }}
 
   data.forEach(function(d) {
     if (d.latitude) {
-      const innerHTML = `
+      const popup_content = `
         <div>
           <ul style="list-style-type:none; padding-left:0px">
-            ${d.isotype ? '<li><b><a href="../strains/isotype/' + d.isotype + '">Isotype: ' + d.isotype + '</a></b></li>': ''}
+            ${d.isotype ? '<li><b><a href="../isotype/' + d.isotype + '">Isotype: ' + d.isotype + '</a></b></li>': ''}

```

Comment:
prefer the backtick string interpolation since it is what we have in other places in the code base.